Mathlib/MeasureTheory/Group/LIntegral.lean

```diff
@@ -53,6 +53,21 @@ theorem lintegral_div_right_eq_self [IsMulRightInvariant μ] (f : G → ℝ≥0
 
 end MeasurableMul
 
+section MeasurableInv
+
+variable [Group G] [MeasurableInv G]
+
+/-- Involuting a function does not change its Lebesgue integral with respect to a
```

Comment:
I have never seen the expression "involuting a function" and did not successfully guess what that was. Can you replace that by a more explicit description?